onebusaway-android/src/main/java/org/onebusaway/android/io/request/bike/OtpBikeStationRequest.java

```diff
@@ -0,0 +1,87 @@
+/*
+* Copyright (C) Rodrigo Carvalho (carvalhorr@gmail.com)
```

Comment:
Please add date to copyright header, like:

~~~
 Copyright (C) 2017 Rodrigo Carvalho (carvalhorr@gmail.com)
~~~